web/src/app/user/user.component.ts

```diff
@@ -57,4 +59,32 @@ export class UserComponent implements OnInit{
       console.log("Logout error", error);
     });
   }
+
+  getUser() {
+    this.userService.getUser(this.user.id)
+        .subscribe((data: FirebaseUserModel) => this.user = { 
+          billingAddress: data['billingAddress'],
+          email: data['email'],
+          firstName: data['firstName'],
+          id: data['id'],
+          lastName: data['lastName'],
+          nickname: data['nickname'],
+          shippingAddress : data['shippingAddress'],
+          provider: this.user.provider,
+          image: this.user.image,
+          name: this.user.name
+        });
+  }
+
+  updateUser() {
+    this.user.name = this.user.firstName + " " + this.user.lastName;
```

Comment:
We should switch this to use JavaScript string templates as it is much more readable.

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)